controller.py

```diff
@@ -69,41 +70,42 @@ def start_new_game():
   (and continuation) of a game of Bites.
   """
   players = prepare_list_of_players()
-  anthill_order = choose_anthill_rule()
-  play_bites = Bites(ANTS, TOKENS_FOR_TRAIL, players, anthill_order)
+  anthill_order = choose_game_rule(ANTHILL_CARD_DICT, PROMPT_TEXT_RULE_CHOICE_ANTHILL)
```

Comment:
Consider renaming `anthill_order` to `anthill_rule` for consistency with the `wine_rule` and future expected `chocolate_rule`.